src/Simplex/Chat.hs

```diff
@@ -5659,11 +5689,11 @@ saveSndChatItem' user cd msg@SndMessage {sharedMsgId} content ciFile quotedItem
     pure ciId
   liftIO $ mkChatItem cd ciId content ciFile quotedItem (Just sharedMsgId) itemTimed live createdAt Nothing createdAt
 
-saveRcvChatItem :: ChatMonad m => User -> ChatDirection c 'MDRcv -> RcvMessage -> UTCTime -> CIContent 'MDRcv -> m (ChatItem c 'MDRcv)
+saveRcvChatItem :: (ChatMonad m, ChatTypeQuotable c) => User -> ChatDirection c 'MDRcv -> RcvMessage -> UTCTime -> CIContent 'MDRcv -> m (ChatItem c 'MDRcv)
```

Comment:
same here, not sure we need to even add constraints to these functions, because it might be simpler to create specific function for local chats, or use `createInternalChatItem`. Local chat items are "snd"/"rcv" only in appearance, they won't really be sent or received, and so they don't require `msg`, `sharedMsgId_` used in saveRcv/SndChatItem functions, meaning these functions won't fit the use case.

Reply:
`getChatItemQuote_` will have to "throw" some error from store operation in IO that isn't gonna even happen